cdap-ui/app/cdap/components/RouteToNamespace/index.js

```diff
@@ -46,23 +46,31 @@ export default class RouteToNamespace extends Component {
     if (!list || list.length === 0) { return; }
 
     /**
-     * 1. Check if localStorage has a 'DefaultNamespace' set by the user, if not,
-     * 2. Check if there is a 'default' namespace from backend, if not,
-     * 3. Take first one from the list of namespaces from backend.
+     * 1. Check if localStorage has an already selected namespace set by the user, if not,
+     * 2. Check if localStorage has a 'DefaultNamespace' set by the user, if not,
```

Comment:
Why do we need to store last accessed namespace? Because of this, the default namespace will never be chosen. It will always go to the last accessed